Emby.Server.Implementations/Services/HttpContextExtension.cs

```diff
@@ -0,0 +1,33 @@
+using MediaBrowser.Model.Services;
+using Microsoft.AspNetCore.Http;
+
+namespace Emby.Server.Implementations.Services
+{
+    /// <summary>
+    /// Extention to enable the service stack request to be stored in the HttpRequest object.
+    /// </summary>
+    public static class HttpContextExtension
+    {
+        private const string SERVICESTACKREQUEST = "ServiceRequestStack";
+
+        /// <summary>
+        /// Set the service stack request.
+        /// </summary>
+        /// <param name="httpContext">The HttpContext instance.</param>
+        /// <param name="request">The IRequest instance.</param>
```

Comment:
Just to be consistent with how this is described in the get method below

```suggestion
        /// <param name="request">The service stack request instance.</param>
```